src/components/Radio/Radio.scss

```diff
@@ -1,26 +1,27 @@
 @use "../../assets/theme/macbis-colors" as c;
 
-.usa-radio__input--tile+[class*=__label] {
+.usa-radio__input--tile + [class*="__label"] {
```

Comment:
Just curious, does SCSS freak out if you don't include these spaces?

Reply:
I didn't intentionally make changes to this file. Prettier must have gotten a hold of that. I can undo this.